docs/BUILD.dpdkstat.md

```diff
@@ -201,15 +201,19 @@ See also: http://dpdk.org/doc/guides/prog_guide/multi_proc_support.html
  *  Generate the build script as specified below. (i.e. run `build.sh`).
  *  Configure collectd with the DPDK shared library:
 
-        ./configure --with-libdpdk=/usr
+        ./configure LIBDPDK_CPPFLAGS="-I/usr/include/dpdk" LIBDPDK_LDFLAGS="-L/usr/lib"
```

Comment:
Correct me if I'm wrong : you mean that setting these flags is not needed if we set it as above since those are default values, right ?
If so, then what about clarifying that if user needs to set custom header/lib dpdk locations, then may set it via these variables,  exactly like in [8259168](https://github.com/collectd/collectd/commit/8259168611932117193bcf891b1c063a619d32e3) ./configure LIBDPDK_CPPFLAGS="-I/path/to/your/libdpdk_headers" LIBDPDK_LDFLAGS="-L/path/to/your/libdpdk.so -Wl,--no-as-needed"

>